### DISCUSSION TITLE
How to install AxelChat on Linux?

### DISCUSSION THREAD

**Discussion by alchem1no** [3dproger/AxelChat#490] [Q&A]
Hi everyone. I have Fedora Linux installed on my 2nd laptop and I tried to setup AxelChat using Bottles. But when i run AxelChat it closes after a couple of seconds.
Can someone help me with it?

**Comment by 3dproger** [comment]
Hi! I'll try to figure out this problem. Were any notifications displayed immediately after closing AxelChat?

**Reply by alchem1no** [reply]
> Hi! I'll try to figure out this problem. Were any notifications displayed immediately after closing AxelChat?

No. It just closes.
It would be great if a web-based version of this chat existed. That would solve my issue immediately.

**Reply by 3dproger** [reply]
> No. It just closes.

Ok, thanks for the info, I'll try to figure this out.

> It would be great if a web-based version of this chat existed. That would solve my issue immediately.

Ok, development is already moving a little in this direction

**Reply by alchem1no** [reply]
Also can I compile AxelChat for Linux by myself?

**Reply by 3dproger** [reply]
> Also can I compile AxelChat for Linux by myself?

Unfortunately, not yet, as I hid the source code because it could be used in bad faith. But maybe in the future I will release builds for Linux or even open the source code

**Reply by 3dproger** [reply]
However, the source code of web widgets is open and you can try to use it: https://github.com/3dproger/AxelChatWidgets

**Comment by SPRATAY** [comment]
![изображение](https://github.com/user-attachments/assets/4cba96fb-0de7-47ae-82ab-22406446a43f)
![изображение](https://github.com/user-attachments/assets/97040f60-06cb-4161-a39b-1a9f5683c201)
![изображение](https://github.com/user-attachments/assets/ce8b63f1-4a24-4ad7-9ef2-e4d362a11d67)

((((( linux so bad

**Comment by 3dproger** [comment]
> ((((( linux so bad

Oh my... I take it you used Wine? I think I'll add this to the issues https://github.com/3dproger/AxelChat/issues/542

**Comment by DenpaBorisu** [comment]
I've used axel chat on wine and it works as good as you could expect, so it fulfills its purpose

**Comment by 3dproger** [comment]
I dare to assume that on different versions of Wine AxelChat works differently
